index.html

```diff
@@ -3484,6 +3484,29 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+
+        <p>
+Additional `application/vc+` media type transformation specifications can be
+defined to provide implementers with further options for producing objects
+compliant with the base media type.
+
+Specifications are registered with the [[?VC-SPECS-DIR]] and
+MUST result in a serialization
+that is objectively testable via a test suite.
+
+Transformations from other media types MUST state the directionallity of the 
+transformation from the options:
+
+<ul>
+  <li>Unidirectional</li>
+  <li>Bidirectional</li>
+</ul>
+
+Bidirectional transformations MUST result in a lossless conversion between 
+the originating data format and the resulting object compliant with the base 
+media type.
```

Comment:
This seems to define Unidirectional, rather than Bidirectional.

Reply:
I feel this is resolved.